apps/progs.h

```diff
@@ -2,7 +2,7 @@
  * WARNING: do not edit!
  * Generated by apps/progs.pl
  *
- * Copyright 1995-2016 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 1995-2017 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
We change it when the file changes.  progs.h is interesting since progs.pl generates it.  That file got a copyright update, so it propagated to progs.h, as part of #3657.  But looking at it now, the script didn't change.  So...  pushing an updated commit to fix this.